<launch>

  <!-- Turtlesim Node-->
  <node pkg="turtlesim" type="turtlesim_node" name="simulator"/>

  <node pkg="turtlesim" type="turtle_teleop_key" name="keyboard" output="screen"/>
  <!-- Axes -->
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>

  <node name="turtle_tf_broadcaster" pkg="study_pkg" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
    <param name="turtle_tf_name" type="string" value="turtle1" />
    <param name="carrot_tf_name" type="string" value="sattelite" />
    <remap from="input_pose" to="turtle1/pose"/>
    </node>
</launch>